edxval/tests/test_api.py

```diff
@@ -2885,6 +2885,43 @@ def test_create_transcript_file(self):
         transcript = api.get_video_transcript_data(video_id=video_id, language_code=language_code)
         self.assertEqual(transcript['content'], expected_transcript_content)
 
+    def test_unpublished_create_transcript_file(self):
+        """
+        Tests that an unpublished transcript file is created correctly.
+
+        Scenario:
+        In a course with unpublished transcript, static_dir directory isn't made
+        so we don't explicitly make static_dir in this test
+        create_transcript_file will make static_dir inside file_system
```

Comment:
don't mention exact function names here. Just give context that directory will be created if not present etc.

Reply:
added an improved docstring.